paddle/fluid/operators/mkldnn/fill_constant_mkldnn_op.cc

```diff
@@ -0,0 +1,137 @@
+/* Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+#include "paddle/fluid/operators/utils.h"
+#include "paddle/fluid/platform/mkldnn_reuse.h"
+
+namespace paddle {
+namespace operators {
+
+using framework::Tensor;
+
+template <typename T>
+class FillConstantMKLDNNHandler
+    : public platform::MKLDNNHandlerNoCachingT<T, dnnl::binary> {
+ public:
+  FillConstantMKLDNNHandler(Tensor* out, const float value, dnnl::engine engine,
+                            platform::Place cpu_place)
+      : platform::MKLDNNHandlerNoCachingT<T, dnnl::binary>(engine, cpu_place) {
+    const auto src0_md =
+        dnnl::memory::desc({out->numel()}, platform::MKLDNNGetDataType<T>(),
+                           dnnl::memory::format_tag::a);
+
+    dnnl::primitive_attr attrs;
+    dnnl::post_ops post_ops;
+
+    post_ops.append_eltwise(1.0f, dnnl::algorithm::eltwise_linear, 0.0f, value);
+    attrs.set_post_ops(post_ops);

```

Comment:
Hi, I would like to know more about this part, talk on Teams. Is it dicussion results of you and Jacek or Joanna ?

Reply:
Hi, I have talked about it with Jacek, so since I may note be available in next 2/3 weeks you can discuss it with him